classes/classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
```

Comment:
- [ ] You are supposed to implement a method that serves as the `belongs-to` side (a student belongs to a classroom). This `classroom=` method should be implemented inside the Student class. Don't forget to expose the `:classroom` variable with the `attr_reader`

here's a possible pseudo code for the classroom= method:

- Define a method classroom= that accepts one parameter classroom.
- Set an instance variable @classroom to the classroom parameter.
- Using the unless statement, check whether the classroom already includes the self (i.e., the current student).
- If classroom does not include self, then push self into the classroom's list of students, using the push method.

Here's a pseudo-code to demonstrate what the method would look like:
```ruby
define method classroom=(classroom)
    set @classroom to classroom
    unless classroom includes self
        add self to classroom's list of students using the push method
    end
end
```

Note that this pseudo code is just an example of how the classroom= method works. You will have to implement the actual code.

Reply:
Thanks @bobb-Rob, I will implement changes right away